token-allocation/README.md

```diff
@@ -25,7 +25,7 @@ The adapter takes the following environment variables:
   - `symbol`: Token symbol
   - `balance` (optional): Token balance. `1e18` by default
   - `decimals` (optional): Token decimals. `18` by default
-  - `currency` (optional). Currency we want the price on. `DEFAULT_CURRENCY` by default
+- `currency` (optional). Currency we want the price on. `DEFAULT_CURRENCY` by default
```

Comment:
I would suggest we rename this as `quote` as this is the standard name for a currency we want to price an asset in, and we already use it throughout the codebase in other adapters.